doc/environment.yml

```diff
@@ -17,30 +14,32 @@ dependencies:
   - ipython
   - pillow
   - mock
-  - cartopy
   - pyepsg
-  - contextily
-  - rasterio
-  - geoplot
   - sphinx-gallery
   - jinja2
   - doc2dash
   - matplotlib-scalebar
   # specify additional dependencies to reduce solving for conda
-  - gdal
-  - libgdal
-  - proj
   - geos
   - nbsphinx
   - jupyter_client
   - ipykernel
   - myst-parser
   - folium
-  - libpysal
   - xyzservices
   - packaging
-  - pointpats
   - pip
   - pip:
       - sphinx-toggleprompt
       - geodatasets
+      # proj/pyproj deps from pip to avoid Proj env warning from conda+readthedocs
+      - pyproj
+      - cartopy
+      - fiona
+      - pyogrio
+      - rasterio
+      - contextily
+      # pyproj deps via geopandas conda-forge, can be moved once pyproj
+      # is no longer a required dep on conda forge
```

Comment:
I don't think this will happen. We have `geopandas-base` on conda-forge that does not pull pyproj but `geopandas` does and I am not aware of discussions that it should change.

Reply:
>We have geopandas-base on conda-forge that does not pull pyproj
I'm not sure that's right - I opened #3293 because as I understand it geopandas-base does require pyproj right now. 

I thought these both depended on `geopandas-base` only, but having a look again it seems they depend on `geopandas` directly, so I'll remove this comment then.